src/main/webapp/assets/loading-indicator.svg

```diff
@@ -0,0 +1 @@
+<svg width="60px"  height="60px"  xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" class="lds-rolling" style="background: none;"><circle cx="50" cy="50" fill="none" ng-attr-stroke="{{config.color}}" ng-attr-stroke-width="{{config.width}}" ng-attr-r="{{config.radius}}" ng-attr-stroke-dasharray="{{config.dasharray}}" stroke="#2c3e50" stroke-width="9" r="40" stroke-dasharray="188.49555921538757 64.83185307179586" transform="rotate(324.641 50 50)"><animateTransform attributeName="transform" type="rotate" calcMode="linear" values="0 50 50;360 50 50" keyTimes="0;1" dur="1.2s" begin="0s" repeatCount="indefinite"></animateTransform></circle></svg>
```

Comment:
I see some ng-attributes here:

- ng-attr-stroke="{{config.color}}" 
- ng-attr-stroke-width="{{config.width}}" 
- ng-attr-r="{{config.radius}}" 
- ng-attr-stroke-dasharray="{{config.dasharray}}"

I'm guessing we copied this from an angular app somewhere. Pretty sure they're not necessary here.

Reply:
Removed.